testdata/errors_transaction_failed.xml

```diff
@@ -6,6 +6,7 @@
       <merchant_message>The payment gateway declined the transaction because the security code (CVV) did not match.</merchant_message>
       <customer_message>The security code you entered does not match. Please update the CVV and try again.</customer_message>
       <gateway_error_code>301</gateway_error_code>
+      <three_d_secure_action_token_id>three_d_secure_action_token_id</three_d_secure_action_token_id>
```

Comment:
Minor nit, but for mock/test data would prefer we use a value representative off what is returned, or at the very least not the field name itself as the value. Otherwise it can get a bit confusing to have the value of the field also be the name of the field.

I pulled this one from the [recurly docs](https://dev.recurly.com/page/recurly-3d-secure-2-integration-guide#section--step-3-process-the-purchase-response) you linked to: `ABCDEFGHIJKL012345`